source/atlas-search.txt

```diff
@@ -90,6 +95,120 @@ following actions:
    Search queries, see :atlas:`Atlas Search Tutorials </atlas-search/tutorials/>`
    in the Atlas documentation.
 
+Atlas Search Metadata
+---------------------
+
+Use the ``searchMeta()`` method to create a
+:manual:`$searchMeta </reference/operator/aggregation/searchMeta/>`
+pipeline stage which returns only the metadata part of the results from
+Atlas full-text search queries.
+
+.. tip:: Only Available on Atlas for MongoDB v4.4.11 and later
+
+   This aggregation pipeline operator is only available
+   on :atlas:`MongoDB Atlas </>` clusters running v4.4.11 and later. For a
+   detailed list of version availability, see the MongoDB Atlas documentation
+   on :atlas:`$searchMeta </atlas-search/query-syntax/#-searchmeta>`.
+
+The following example shows the ``count`` metadata for an Atlas search
+aggregation stage:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/builders/AggregateSearchBuilderExample.java
+   :start-after: // begin atlasSearchMeta
+   :end-before: // end atlasSearchMeta
+   :language: java
+   :dedent:
+
+Learn more about this helper from the
+`searchMeta() API documentation <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/Aggregates.html#searchMeta(com.mongodb.client.model.search.SearchCollector)>`__.
+
+.. _java-atlas-search-helpers:
+
+SearchOperator Helper Methods
+-----------------------------
+
+To make building pipeline stages easier, the {+driver-short+} provides helper methods
+for the following operations:
```

Comment:
S: I think we should avoid specifying things as "easier" where we can since "easier" is subjective
```suggestion
The {+driver-short+} provides helper methods
for the following operations:
```